test/rest-api/body/body-json.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'path'
+import { runBrowserWith } from '../../support/runBrowserWith'
+
+test('responds with a JSON response body', async () => {
+  const runtime = await runBrowserWith(
+    path.resolve(__dirname, 'body-json.mocks.ts'),
+  )
+
+  const res = await runtime.request({
+    url: runtime.makeUrl('/json'),
+  })
+  const headers = res.headers()
+  const json = await res.json()
+
+  expect(headers).toHaveProperty('content-type', 'application/json')
```

Comment:
Nit, should these be asserted before the content is read?
Otherwise, I think the behavior would be that `res.json` throws.

Reply:
I don't mind changing the assertions order. Thank you!